content/en/tracing/profiling/java.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Profiling Java Applications
+kind: documentation
+further_reading:
+    - link: 'https://www.datadoghq.com/blog/introducing-datadog-profiling/'
+      tags: 'Blog'
+      text: 'Introducing always-on production profiling in Datadog.'
+---
+
+The Datadog Profiling library supports the Java JRE 11 and higher of both Oracle JDK and OpenJDK. Early-access versions of Java aren't supported.
+Java profiling and tracing are shipped within the same `dd-java-agent.jar` library. In order to use tracing or profiling, you must enable these functions. To enable tracing, follow the [enable tracing documentation][1].
```

Comment:
Also azul 8 and oracle 8 w the commercial flag.

Do we need to specify jre vs jdk? Users might think it applies to one and not the other.

Reply:
I don't have the technical depth to understand this feedback, what would be your phrasing proposal ?

Reply:
Sorry! I originally wrote this feedback assuming is was for Hugo :-)

I think I changed my mind about mention flight recorder in this context :-) My suggestion is `The Datadog Profiler requires Java Flight Recorder. This is supported in OpenJDK 11, Oracle Java 8.TODO with a commercial license, Oracle Java 11, and Zulu Java 8.TODO.`

@jbachorik do you know the exact versions for ^?

> Java profiling and tracing are shipped within the same `dd-java-agent.jar` library. In order to use tracing or profiling, you must enable these functions. To enable tracing, follow the [enable tracing documentation][1].

Tracing is on by default.

Can we link straight to getting started rather than requiring an extra hop through a footnote?